pkgs/development/python-modules/sqlalchemy-jsonfield/default.nix

```diff
@@ -1,20 +1,39 @@
-{ lib, buildPythonPackage, fetchPypi, sqlalchemy, setuptools-scm, setuptools }:
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, sqlalchemy
+, setuptools-scm
+, setuptools
+, git
+, tox
+, sphinx
+, pytest
+, pytest-cov
+, pytest-html
+, pytest-sugar
+, coverage
+, pymysql
+, psycopg2 }:
 
 buildPythonPackage rec {
   pname = "sqlalchemy-jsonfield";
   version = "1.0.0";
 
-  src = fetchPypi {
-    pname = "SQLAlchemy-JSONField";
-    inherit version;
-    sha256 = "12z9mxaphvmsjlxawyxkhxcrc8grhxcrgygsrikkzxgbpljhnvbn";
+  src = fetchFromGitHub {
+    owner = "penguinolog";
+    repo = "sqlalchemy_jsonfield";
+    rev = version;
+    sha256 = "1g5hwcnx7w7vv2021kfcsrnaflpvx6jbyvs6g5p185aipiq3j575";
+    leaveDotGit = true;
   };
 
-  buildInputs = [ setuptools-scm ];
+  nativeBuildInputs = [ git setuptools-scm ];
```

Comment:
Can we use ``SETUPTOOLS_SCM_PRETEND_VERSION = "v${version}";`` instead of git and leaving .git?

Reply:
This works, but the tests require git anyways. I can move git from `nativeBuildInputs` to `checkInputs`, I guess.